certbot/display/ops.py

```diff
@@ -212,11 +212,13 @@ def success_installation(domains):
 
     """
     z_util(interfaces.IDisplay).notification(
-        "Congratulations! You have successfully enabled {0}{1}{1}"
+        "Congratulations! You have successfully {3}ed a certificate for{1}"
```

Comment:
How about "enabled HTTPS" or "installed a certificate and enabled HTTPS"?

Since turning on HTTPS may be the conceptually important point for many users.